snapcraft/internal/pluginhandler.py

```diff
@@ -756,7 +756,8 @@ def _generate_include_set(directory, includes):
     include_files = set()
     for include in includes:
         if '*' in include:
-            matches = glob.glob(os.path.join(directory, include))
+            pattern = os.path.join(directory, include)
+            matches = iglob(pattern, recursive=True)
```

Comment:
Which means you evaluate them all immediately anyway, right? I don't see the difference between this and `glob` for our usage anyway. That said, I don't have a problem with `iglob`, I was just curious for the reasoning behind the switch.